spark-cassandra-connector/src/test/scala/com/datastax/spark/connector/ColumnSelectorSpec.scala

```diff
@@ -31,12 +31,26 @@ class ColumnSelectorSpec extends WordSpec with Matchers {
       columns.map(_.columnName) should be equals Seq("c1", "c3", "c5")
     }
 
+    "return selections with function calls" in {
```

Comment:
@alexliu68 Tests as requested: Here is tested whether the columns required by the function call are checked for existence.